### PR TITLE
#662 Bug fix when opening multiple curves htmls at once

### LINKED ISSUE #662: Issue with command jobs-with-curves when jobs file contains multiple job objects
Issue with command jobs-with-curves: when jobs file contains multiple job objects, only the first set of curves is open in browser. The fix is to open a new browser window for each call of jobs-with-curves, a tab containing a curve set.

### PR DESCRIPTION
**Pull request by marcochiaramello**
Closes #662

### CODE REVIEW

File: util/curvesToHtml/curvesToHtml.py [hunk 1]
```diff
@@ -67,8 +67,10 @@ def readCurvesToHtml(jobs_file, withoutOffset, showpoints, htmlBrowser):
     # ------------------------------------------------#
 
     # parse all jobs in the input file
+    htmlFileList=[]
     for job in jobs_root.iter(namespaceDYN("job")):
 
+
```
Comment (rosiereflo) [inline util/curvesToHtml/curvesToHtml.py]: Please remove useless empty line

File: util/curvesToHtml/curvesToHtml.py [hunk 2]
```diff
@@ -125,21 +127,22 @@ def readCurvesToHtml(jobs_file, withoutOffset, showpoints, htmlBrowser):
...
-                            raise os_error
                     print("... End of generating curves visualization page for job <" + current_job.jobName_ + ">")
 
                 except Exception as exc:
                     print("Fail to generate curves visualization : " + str(exc))
                     sys.exit(1)
 
+    allCurves = ""
+    for ht in htmlFileList:
+        allCurves+=""+ht+" "
```
Comment (rosiereflo) [inline util/curvesToHtml/curvesToHtml.py]: Why removing the use of webbrowser?

Comment (marcochiaramello) [reply util/curvesToHtml/curvesToHtml.py]: As it seems to be at the root of the problem, it does not manage correctly the opening of multiple tabs, in particular if the browser is not running before

File: util/curvesToHtml/curvesToHtml.py (hunk 1 above)
Comment (marcochiaramello) [reply util/curvesToHtml/curvesToHtml.py]: Done

File: util/curvesToHtml/curvesToHtml.py (hunk 2 above)
Comment (rosiereflo) [reply util/curvesToHtml/curvesToHtml.py]: ok with me, I just hope this new solution works with Apple

**Pull request merged**